CMakeLists.txt

```diff
@@ -281,3 +281,7 @@ if (BUILD_TESTING AND EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/perftests")
   add_subdirectory(perftests)
 endif ()
 # vim: ts=4 sw=4 sts=4 et
+
+export(TARGETS ${GDAL_LIB_TARGET_NAME} 
```

Comment:
This should rather be moved to gdal.cmake close to lines that do similar things (line 546-563)